### ISSUE TITLE
`jax.tree_util.tree_multimap` deprecated; `stax` changed location

### ISSUE THREAD

**Issue by UltimateJupiter** [miguel-rg/dynamical-loss-functions#2]
In the latest version of JAX on google colab, `jax.tree_util.tree_multimap` is deprecated, it seems that changing to `jax.tree_util.tree_map` solves the issue.

Also `stax` is now moved to `jax.example_libraries` instead of `jax.experimental`.

**Comment by miguel-rg** [comment]
Dear @UltimateJupiter,

So sorry that I forgot about this. It is fixed now using your suggestions, thank you very much.

Please, let me know if I can help with anything else,

Best,
Miguel
